content/en/blog/2023-12-12-announcing-flux-v2.2.0/index.md

```diff
@@ -0,0 +1,149 @@
+---
+author: hiddeco
+date: 2023-12-12 16:00:00+00:00
+title: Announcing Flux 2.2 GA
+description: "Flux v2.2.0: Improving your Helm experience!"
+url: /blog/2023/12/flux-v2.2.0/
+tags: [announcement]
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+---
+
+We are thrilled to announce the release of [Flux v2.2.0](https://github.com/fluxcd/flux2/releases/tag/v2.2.0)! In this post, we will highlight some of the new features and improvements included in this release, with the primary theme as the (many) changes made to the [helm-controller](https://fluxcd.io/flux/components/helm/).
+
+## Important things first: API changes
+
+### Static Objects
+
+The **Alert v1beta3**, **Provider v1beta3**, and **HelmChart** of type `oci`  APIs are now static objects. This means that they are no longer reconciled by a reconciler and will have no status field. A custom resource of these types is considered ready once it is created.
+
+Existing objects will undergo a one-time auto-migration to remove the status field.
+
+### OCIRepository v1beta2 and HelmChart v1beta2
+
+The OCIRepository and HelmChart OCI was extended with the following fields:
+
+- `.spec.matchOIDCIdentity[].issuer` allows you to specify a regex pattern to match against the issuer of the certificate related to the artifact signature used for OCI keyless verification.
+- `.spec.matchOIDCIdentity[].subject` allows you to specify a regex pattern to match against the subject of the certificate related to the artifact signature used for OCI keyless verification.
+
+### Bucket v1beta2
+
+- A new field, `.spec.prefix`, has been added which enables server-side filtering of files if the object's .spec.provider is set to generic/aws/gcp.
+
+### HelmRepository v1beta2 and ImageRepository v1beta2
+
+- A new boolean field, `.spec.insecure`, has been introduced which allows connecting to a non-TLS HTTP container registry. For HelmRepository API, it is only considered if the object's `.spec.type` is set to `oci`.
```

Comment:
Lists do not work nicely when the lists are short and you want to maintain a reader their attention. Should rewrite this to be fluent.